plugins/source-kafka/pkg/factories/command_factory_test.go

```diff
@@ -0,0 +1,71 @@
+// Copyright © 2020 The Knative Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package factories
+
+import (
+	"testing"
+
+	"gotest.tools/assert"
+	"knative.dev/client-contrib/plugins/source-kafka/pkg/types"
+)
+
+func TestNewKafkaSourceCommandFactory(t *testing.T) {
+	commandFactory := createKafkaSourceCommandFactory()
+
+	assert.Assert(t, commandFactory != nil)
+}
+
+func TestCommand_KnSourceFactory(t *testing.T) {
+	commandFactory := createKafkaSourceCommandFactory()
+
+	assert.Assert(t, commandFactory.KnSourceFactory() != nil)
+}
+
+func TestSourceCommand(t *testing.T) {
+	commandFactory := createKafkaSourceCommandFactory()
+
+	assert.Assert(t, commandFactory.SourceCommand() != nil)
+}
+
+func TestCreateCommand(t *testing.T) {
+	commandFactory := createKafkaSourceCommandFactory()
+
+	assert.Assert(t, commandFactory.CreateCommand() != nil)
+}
+
+func TestDeleteCommand(t *testing.T) {
```

Comment:
I wonder if we should add more to the basic tests to include checks for params...?